playground-common/src/PSGenerator/Common.hs

```diff
@@ -160,6 +160,11 @@ scientificBridge = do
     typeModule ^== "Data.Scientific"
     pure psNumber
 
+ratioBridge :: BridgePart
+ratioBridge = do
+    typeName ^== "Ratio"
+    typeModule ^== "Data.Ratio"
```

Comment:
Should be `PlutusTx.Ratio` (we had to redefine it to make it work with the plugin)

Reply:
Yep that's what the error in the issue is about